server/manager/src/main/java/org/apache/accumulo/manager/compaction/coordinator/CompactionCoordinator.java

```diff
@@ -1118,6 +1083,137 @@ private void cleanUpCompactors() {
     }
   }
 
+  private Set<CompactorGroupId> getCompactionServicesConfigurationGroups()
```

Comment:
This method is similar, but not exactly the same, as CheckCompactionConfig.validate.